tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -36,12 +39,14 @@ class TelemetryValidator(object):
 
   def __init__(self, entities, timeout, callback):
     super().__init__()
-    self.entities = entities
+    self.entities = dict(filter((lambda e: e[1].translation),
```

Comment:
this feels like a magic number to me.  #Define?

Reply:
entities: a dict with entity_name as a key and EntityIntance as value.
e[1] is kist accessing the entityInstance obj

Reply:
got it.  The internet did not yield a more elegant way to do it.  

It would be clearer, however if the varname was more explicit about what it was:  ex: entities_with_translations

Reply:
The technique explained here seems more readable:

https://stackoverflow.com/questions/23862406/filter-items-in-a-python-dictionary-where-keys-contain-a-specific-string

Reply:
updated to entities_with_translations, I prefer the lambda one :)